frontend/src/upload_file.ts

```diff
@@ -281,7 +305,8 @@ class UploadFile {
     const upload = this.uploads[uploadIndex];
 
     this.renderer.deleteFile(uploadIndex);
-    delete this.uploads[uploadIndex];
+    // delete this.uploads[uploadIndex];
+    this.uploads.splice(uploadIndex, 1)
```

Comment:
@jma7 This looks suspicious. Please double check.

Reply:
Fixed.